tests/test_functions.py

```diff
@@ -159,78 +152,103 @@ def test_clean_names_original_columns(dataframe):
 
 
 def test_remove_empty(null_df):
-    df = remove_empty(null_df)
+    df = null_df.remove_empty()
     assert df.shape == (8, 2)
 
 
 def test_get_dupes():
     df = pd.DataFrame()
     df["a"] = [1, 2, 1]
     df["b"] = [1, 2, 1]
-    df_dupes = get_dupes(df)
+    df_dupes = df.get_dupes()
     assert df_dupes.shape == (2, 2)
 
     df2 = pd.DataFrame()
     df2["a"] = [1, 2, 3]
     df2["b"] = [1, 2, 3]
-    df2_dupes = get_dupes(df2)
+    df2_dupes = df2.get_dupes()
     assert df2_dupes.shape == (0, 2)
 
 
-def test_encode_categorical():
-    df = pd.DataFrame()
-    df["class_label"] = ["test1", "test2", "test1", "test2"]
-    df["numbers"] = [1, 2, 3, 2]
-    df = encode_categorical(df, "class_label")
+def categoricaldf_strategy():
```

Comment:
`categoricaldf_strategy`

Kind of insignificant comment, but maybe rename to `categorical_df_strategy`? 🤔